wolfssl/openssl/evp.h

```diff
@@ -429,6 +429,8 @@ struct WOLFSSL_EVP_CIPHER_CTX {
     int     gcmBufferLen;
     byte*   gcmAuthIn;
     int     gcmAuthInSz;
+    int     gcmIvGenEnable;
```

Comment:
Should these be bit-fields `:1`? Use unsigned value if changing and place at end of struct.

Reply:
Yeah, good idea. Will change.